src/briefcase/commands/run.py

```diff
@@ -34,10 +34,10 @@ class LogFilter:
         r"\n(?P<a2> )?\n(?P<a3> \n)?FAILED( \(.*\))?$)"
         # Pytest
         r"|"
-        r"(^={10,} ("
-        r"(\d+ failed(, \d+ passed)?(, \d+ skipped)?(, \d+ errors?)?)"
-        r"|(\d+ errors?)"
-        r") in \d+.\d+s ={10,}$)"
+        r"(^={1,} ("
+        r"((\d+ failed)((, )?\d+ (passed|skipped|deselected|xfailed|xpassed|warnings?|errors?))*)"
+        r"|(((, )?\d+ (failed|passed|skipped|deselected|xfailed|xpassed|warnings?|errors?))*((, )?\d+ errors?))"
```

Comment:
That's the idea; but also capturing "there *must* be one of either failed or errors"; otherwise, a success result can easily match the failure criteria.